Aztec/Classes/TextKit/ParagraphProperty/TextList.swift

```diff
@@ -14,14 +15,51 @@ open class TextList: ParagraphProperty {
         case ordered
         case unordered
 
-        func markerText(forItemNumber number: Int) -> String {
+        func markerText(forItemNumber number: Int, indentLevel: Int? = nil) -> String {
             switch self {
-            case .ordered:      return "\(number)."
-            case .unordered:    return "\u{2022}"
+            case .ordered:
+                if indentLevel == nil {
+                    return "\(number)."
+                }
+                
+                switch indentLevel {
+                case 1:
+                    return "\(number)."
+                case 2:
+                    let text = getLetter(for: number)
```

Comment:
What do you think about using `NSTextList(markerFormat: .lowercaseAlpha, options: 0)` here so we offload that part to [UIKit](https://developer.apple.com/documentation/uikit/nstextlist)? We'd get the string via `marker(forItemNumber: number)`.

Reply:
Tried it, but it doesn't match the HTML spec behavior: https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element

Namely, after the 27th element, it should concatenate the letters, then traverse them. So after `z` we get `aa`, then `ab`, etc. The algorithm I implemented reproduces the HTML standard.